ast-factory/pom.xml

```diff
@@ -3,15 +3,15 @@
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
     <parent>
         <groupId>com.thatdot</groupId>
-        <artifactId>front-end-parent-9.1.0</artifactId>
```

Comment:
Having the version number here in the artifactId seems silly - that's what the version below is for.
Upstream on Maven Central is is `front-end-parent-9.0` - https://search.maven.org/artifact/org.opencypher/front-end-parent-9.0
Following that, we could rename this to `front-end-parent-9.1`? Though why does any version number need to be in the artifactId - why not just `front-end-parent`?

Reply:
I'm happy to change it as part of this PR if that would be helpful. The meat of the PR is more the dependency scoping changes.

Reply:
It's fine for now since the work is already done - just next time we touch this we could remove the version number from the jar?

Reply:
That sounds a lot better! Now that I know I will remove those extraneous versions from the string next time we touch this.